<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthorisationWindow</class>
 <widget class="QMainWindow" name="AuthorisationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>505</width>
    <height>486</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="username_line_edit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_line_edit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="sign_in_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign in</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sign_up_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
   <widget class="QLabel" name="username_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="password_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="welcome_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>411</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Welcome to Nyakutka's Brewery!</string>
    </property>
   </widget>
   <widget class="QLabel" name="sign_up_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>340</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Don't have an account?</string>
    </property>
   </widget>
   <widget class="QLabel" name="invalid_auth_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Invalid username or password</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
